tests/tests_unit.yml

```diff
@@ -29,15 +29,24 @@
 - hosts: all
   name: execute python unit tests
   tasks:
-    - name: Copy python modules
-      copy:
-        src: "{{ item }}"
-        dest: /tmp/test-unit-1/
-        local_follow: false
+    - name: Check python modules existence
+      local_action: stat path="{{ item }}"
+      register: _present_modules
       loop:
         - ../library/network_connections.py
         - unit/test_network_connections.py
         - ../module_utils/network_lsr
+        - module_utils/network_lsr
+        - modules/network_connections.py
```

Comment:
what is the path for these?  That is, in which directory is this task assuming it is being run from?

Reply:
In the `tests` directory. There are symlinks `modules` targeting `../library` and `module_utils` targeting `../module_utils`. When converting to the collections format, the symlinks are replaced with the real dirs and files to avoid having symlinks in the collections format.